thirdeye/thirdeye-frontend/app/pods/rootcause/route.js

```diff
@@ -342,6 +342,9 @@ export default Route.extend(AuthenticatedRouteMixin, {
       }
     }
 
+    // Add maxTime
+    context.maxTime = metricEntity.attributes.maxTime[0];
```

Comment:
metricEntity is not set when coming from anomalyId or sessionId.
(Also this doesn't belong into the context, i.e. it's not part of the RCA "search query")

Reply:
let me remove this if we decided to go with the max time being  EOD today.